x/subscription/types/cronspec.go

```diff
@@ -384,7 +384,8 @@ func (spec CompiledCronSpec) RoundUp(ts int64, tzoffset int32) int64 {
 
 	Success:
 		// Check weekday
-		if !spec.Wday.Test(uint32(Weekday(tm.Year, tm.Month, tm.Mday))) {
+		result, error := spec.Wday.Test(uint32(Weekday(tm.Year, tm.Month, tm.Mday)))
+		if error == nil && !result {
```

Comment:
Test error looks exceptional case,
how about this?
--------------------------
result, error := spec.Wday.Test(uint32(Weekday(tm.Year, tm.Month, tm.Mday)))
assert(error==nil);
 		if !result {

Reply:
Plus some "recovery": https://github.com/cosmos/cosmos-sdk/blob/master/docs/core/runtx_middleware.md ? disable the module?